chaosreliably/types.py

```diff
@@ -21,3 +35,70 @@ class ObjectiveResult(BaseModel):
 
     def parse_list(obj: Any) -> "List[ObjectiveResult]":
         return parse_obj_as(List[ObjectiveResult], obj)
+
+
+class ChaosToolkitType(Enum):
+    EXPERIMENT: str = "Chaos Toolkit Experiment"
+    EXPERIMENT_EVENT: str = "Chaos Toolkit Experiment Event"
+    EXPERIMENT_RUN: str = "Chaos Toolkit Experiment Run"
+    EXPERIMENT_VERSION: str = "Chaos Toolkit Experiment Version"
+
+
+class EntityContextExperimentLabels(BaseModel):
+    type: str = Field(
+        default=ChaosToolkitType.EXPERIMENT.value, alias="ctk_type", const=True
+    )
+    title: str = Field(alias="ctk_experiment_title")
+
+
+class EntityContextExperimentVersionLabels(BaseModel):
+    type: str = Field(
+        default=ChaosToolkitType.EXPERIMENT_VERSION.value, alias="ctk_type", const=True
+    )
+    commit_hash: str = Field(alias="ctk_commit_hash")
+    source: HttpUrl = Field(alias="ctk_source")
+
+
+class EntityContextExperimentRunLabels(BaseModel):
+    type: str = Field(
+        default=ChaosToolkitType.EXPERIMENT_RUN.value, alias="ctk_type", const=True
+    )
+    id: UUID4 = Field(default=uuid4(), alias="ctk_run_id", const=True)
```

Comment:
won't the default be the same for all instances of that type? shouldn't this be:

```
id: UUID4 = Field(default=lambda: uuid4(), alias="ctk_run_id", const=True)
```

or maybe:

```
id: UUID4 = Field(default=uuid4, alias="ctk_run_id", const=True)
```

Reply:
Yep!